integration-tests/src/test/resources/tunneling/JmsTestClient.java

```diff
@@ -0,0 +1,122 @@
+import java.util.Hashtable;

```

Comment:
missing copyright

Reply:
Copyright (c) 2020, Oracle Corporation and/or its affiliates.
Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.